test/test_blocks_on_demand.py

```diff
@@ -362,3 +364,35 @@ def test_endpoint_if_some_pending():
     deploy(CONTRACT_PATH, inputs=["10"])
     resp = demand_block_creation()
     _assert_correct_block_creation_response(resp)
+
+
+@devnet_in_background("--blocks-on-demand")
+def test_increase_time_in_block_on_demand_mode():
+    """Test block creation with increase_time and pending txs"""
+    deploy_info = deploy(CONTRACT_PATH, inputs=["0"])
+    assert_tx_status(deploy_info["tx_hash"], "PENDING")
+    latest_block_timestamp = get_block(block_number="latest", parse=True)["timestamp"]
+
+    # increase_time which generates a new block with pending txs
+    increase_time_response = increase_time(10000)
+
+    latest_block = get_block(block_number="latest", parse=True)
+    assert latest_block["timestamp"] >= latest_block_timestamp + 10000
+    assert latest_block["block_hash"] == increase_time_response.json()["block_hash"]
+    assert_tx_status(deploy_info["tx_hash"], "ACCEPTED_ON_L2")
+
+
+@devnet_in_background("--blocks-on-demand")
+def test_set_time_in_block_on_demand_mode():
+    """Test block creation with set_time and pending txs"""
+    deploy_info = deploy(CONTRACT_PATH, inputs=["0"])
+    assert_tx_status(deploy_info["tx_hash"], "PENDING")
+    latest_block_timestamp = get_block(block_number="latest", parse=True)["timestamp"]
+
+    # set_time which generates a new block with pending txs
+    set_time_response = set_time(latest_block_timestamp + 10000)
+
+    latest_block = get_block(block_number="latest", parse=True)
+    assert latest_block["timestamp"] >= latest_block_timestamp + 10000
```

Comment:
Why not use `==` instead of `>=`?

Reply:
I'll check it, `set_time` should work with `==`

Reply:
fixed in cdc0929b0636f7660ce46af90f420701daac4f8b